### DISCUSSION TITLE
sly completion in company mode - how to hide percentage match score?

### DISCUSSION THREAD

**Discussion by nicholascarroll** [joaotavora/sly#499] [Q&A]
In Sly company-mode there is a percentage score that tells you how well the completion candidate matched. How to always hide this?

**Comment by joaotavora** [comment]
Good question, but not sure at the moment.  Could probably be done at the company level.

**Comment by nicholascarroll** [comment]
I'll use the simple completions for now. thanks for Sly it is great.
